File: java/Samples/Biometrics/VB/NTemplateSampleVB/NewNERecordForm.vb
Imports System.Windows.Forms

Partial Public Class NewNERecordForm
	Inherits Form
	Public Sub New()
		InitializeComponent()
	End Sub

	Public Property RecordWidth() As UShort
		Get
			Return CUShort(nudWidth.Value)
		End Get
		Set(ByVal value As UShort)
			nudWidth.Value = value
		End Set
	End Property

	Public Property RecordHeight() As UShort
		Get
			Return CUShort(nudHeight.Value)
		End Get
		Set(ByVal value As UShort)
			nudHeight.Value = value
		End Set
	End Property
End Class

File: java/Samples/Biometrics/CS/NTemplateSampleCS/NewNERecordForm.cs
using System.Windows.Forms;

namespace Neurotec.Samples
{
	public partial class NewNERecordForm : Form
	{
		public NewNERecordForm()
		{
			InitializeComponent();
		}

		public ushort RecordWidth
		{
			get
			{
				return (ushort)nudWidth.Value;
			}
			set
			{
				nudWidth.Value = value;
			}
		}

		public ushort RecordHeight
		{
			get
			{
				return (ushort)nudHeight.Value;
			}
			set
			{
				nudHeight.Value = value;
			}
		}
	}
}
